storage/system-test/acl.test.js

```diff
@@ -13,225 +13,92 @@
 
 'use strict';
 
-var path = require('path');
-var Storage = require('@google-cloud/storage');
-var uuid = require('node-uuid');
-var program = require('../acl');
-
-var storage = Storage();
-var bucketName = 'nodejs-docs-samples-test-' + uuid.v4();
-var fileName = 'test.txt';
-var entity = 'allAuthenticatedUsers';
-var role = 'READER';
-var filePath = path.join(__dirname, '../resources', fileName);
-
-var expected = {
-  entity: entity,
-  role: role
-};
-
-describe('storage:acl', function () {
-  before(function (done) {
-    storage.createBucket(bucketName, function (err, bucket) {
-      if (err) {
-        return done(err);
-      }
+const storage = require(`@google-cloud/storage`)();
+const uuid = require(`node-uuid`);
+const path = require(`path`);
+const run = require(`../../utils`).run;
+
+const cwd = path.join(__dirname, `..`);
+const bucketName = `nodejs-docs-samples-test-${uuid.v4()}`;
+const userEmail = `00b4903a973860a50828a620e09dde96aae6122ca4f0835bd469384659f1a5b8`;
+const fileName = `test.txt`;
+const filePath = path.join(__dirname, `../resources`, fileName);
+const cmd = `node acl.js`;
+
+describe('storage:acl', () => {
+  before((done) => {
+    storage.createBucket(bucketName, (err, bucket) => {
+      assert.ifError(err);
       bucket.upload(filePath, done);
     });
   });
 
-  after(function (done) {
-    storage.bucket(bucketName).deleteFiles({ force: true }, function (err) {
-      if (err) {
-        return done(err);
-      }
-      storage.bucket(bucketName).delete(done);
+  after((done) => {
+    storage.bucket(bucketName).file(fileName).delete((err) => {
+      assert.ifError(err);
+      setTimeout(() => storage.bucket(bucketName).delete(done), 2000);
```

Comment:
I take it you want this to return an error if the bucket `delete()` operation fails?